homeassistant/components/suez_water/sensor.py

```diff
@@ -84,26 +86,49 @@ def icon(self):
 
     def _fetch_data(self):
         """Fetch latest data from Suez."""
-        from pysuez.client import PySuezError
-
         try:
             self.client.update()
-
+            # _state holds the volume of consumed water during previous day
             self._state = self.client.state
-            self._attributes = self.client.attributes
             self.success = True
+            self._available = True
+            self._attributes[
+                'attribution'] = self.client.attributes[
+                    'attribution']
+            self._attributes['thisMonthConsumption'] = {}
+            for item in self.client.attributes['thisMonthConsumption']:
+                self._attributes['thisMonthConsumption'][
+                    item] = self.client.attributes[
+                        'thisMonthConsumption'][item]
+            self._attributes['previousMonthConsumption'] = {}
+            for item in self.client.attributes['previousMonthConsumption']:
+                self._attributes['previousMonthConsumption'][
+                    item] = self.client.attributes[
+                        'previousMonthConsumption'][item]
+            self._attributes[
+                'highestMonthlyConsumption'] = self.client.attributes[
+                    'highestMonthlyConsumption']
+            self._attributes[
+                'lastYearOverAll'] = self.client.attributes[
+                    'lastYearOverAll']
+            self._attributes[
+                'thisYearOverAll'] = self.client.attributes[
+                    'thisYearOverAll']
+            self._attributes['history'] = {}
+            for item in self.client.attributes['history']:
+                self._attributes[
+                    'history'][item] = self.client.attributes[
```

Comment:
What are the items?

Reply:
Latest months consumption.
I can't know the number of items in advance.

Reply:
Is each item key the name of the month and the item value is the month consumption in liters?

Please describe the item in more detail.

Reply:
Sorry, did not see this question.
`history` will looks like that:
```
{ "Février 2019": 10109, "Mars 2019": 7553, "Avril 2019": 9003, "Mai 2019": 8844, "Juin 2019": 10472, "Juillet 2019": 4168 }
```
It seems that history contains last 6 months consumption.